Sources/RoutingKit/TrieRouter.swift

```diff
@@ -112,8 +113,9 @@ public final class TrieRouter<Output>: Router, CustomStringConvertible {
         if let output = currentNode.output {
             // return the currently resolved responder if there hasn't been an early exit.
             return output
-        } else if let catchall = currentCatchall {
+        } else if let (catchall, matched) = currentCatchall {
             // fallback to catchall output if we have one
+            parameters.set("**", to: matched)
```

Comment:
I wonder if it would make more sense to store this in a separate property like `parameters.catchAll`? Although I do like the simplicity of just using `**`. One benefit of a separate property is we could store `[String]` directly instead of needing to rejoin the slices on `/` first.

Reply:
Then I should set `parameters.catchAll` to be `String?` right? or as `String` with `""` as default?